### PR TITLE
Feat/husky

### PR DESCRIPTION
**Pull request by kwiss**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,12 +6,25 @@
...
+    "@commitlint/config-conventional": "^8.0.0",
+    "husky": "^2.4.1"
   },
   "dependencies": {
     "lerna": "^3.15.0"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint",
+      "pre-push": "yarn test",
```
Comment (douglasduteil) [inline package.json]: this will require a db de run...

Comment (kwiss) [reply package.json]: yes let's remove that hook

File: package.json [hunk 2]
```diff
@@ -1,16 +1,27 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "yarn build --stream --ignore @{emjpm,optional}/*",
+    "postinstall": "yarn build --stream --ignore @{emjpm,optional}/* && node node_modules/husky/lib/installer/bin install",
```
Comment (douglasduteil) [inline package.json]: this is strange ...

Comment (kwiss) [reply package.json]: i've published a new version with script install split in an install script
You can see in this issue that it's a yarn related problem :
https://github.com/typicode/husky/issues/227

**Pull request merged**